go/ct/spc/specification.go

```diff
@@ -719,6 +719,129 @@ func getAllRules() []Rule {
 		effect:    NoEffect().Apply,
 	})...)
 
+	// --- TLOAD ---
+
+	rules = append(rules, rulesFor(instruction{
+		op:        TLOAD,
+		staticGas: 100,
+		pops:      1,
+		pushes:    1,
+		parameters: []Parameter{
+			StorageAccessKeyParameter{},
+		},
+		conditions: []Condition{
+			RevisionBounds(R13_Cancun, NewestSupportedRevision),
+			IsTransientSet(Param(0)),
+		},
+		effect: func(s *st.State) {
+			key := s.Stack.Pop()
+			value := s.Transient.GetStorage(key)
+			s.Stack.Push(value)
+		},
+	})...)
+
+	rules = append(rules, rulesFor(instruction{
+		op:        TLOAD,
+		staticGas: 100,
+		pops:      1,
+		pushes:    1,
+		parameters: []Parameter{
+			StorageAccessKeyParameter{},
+		},
+		conditions: []Condition{
+			RevisionBounds(R13_Cancun, NewestSupportedRevision),
+			IsTransientNotSet(Param(0)),
+		},
+		effect: func(s *st.State) {
+			key := s.Stack.Pop()
+			value := s.Transient.GetStorage(key)
```

Comment:
No need to look up the storage. The result must be zero. You can push zero straight on the stack to emphasis this.